### PR TITLE
update color calendar, ui header

### PR DESCRIPTION
**Pull request by vietbq**

### CODE REVIEW

File: app/assets/javascripts/user.js [hunk 1]
```diff
@@ -60,4 +60,28 @@ $(document).on('page:change', function() {
...
+
+  $('#header-avatar').click(function() {
+    var position = $('#header-avatar').offset();
+    $('#sub-menu-setting').css({'top': position.top + 46, 'left': position.left - 110});
+    $('#prong-header').css({
+      'top': -9,
+      'left': 130,
+      'transform': 'rotateX(' + 180 + 'deg)'
+    });
+    if ($('#sub-menu-setting').hasClass('sub-menu-visible')){
```
Comment (limkimhuor) [inline app/assets/javascripts/user.js]: space before {

File: app/assets/javascripts/user.js [hunk 2]
```diff
@@ -60,4 +60,28 @@ $(document).on('page:change', function() {
       checkbox.push($(this).val());
     });
   }
+
+  $(document).click(function(e) {
+    if ($(event.target).closest('#header-avatar').length == 0){
```
Comment (limkimhuor) [inline app/assets/javascripts/user.js]: space before {

File: app/assets/javascripts/user.js [hunk 3]
```diff
@@ -60,4 +60,28 @@ $(document).on('page:change', function() {
...
+    $('#sub-menu-setting').css({'top': position.top + 46, 'left': position.left - 110});
+    $('#prong-header').css({
+      'top': -9,
+      'left': 130,
+      'transform': 'rotateX(' + 180 + 'deg)'
+    });
+    if ($('#sub-menu-setting').hasClass('sub-menu-visible')){
+      $('#sub-menu-setting').removeClass('sub-menu-visible');
+      $('#sub-menu-setting').addClass('sub-menu-hidden');
+    } else{
```
Comment (limkimhuor) [inline app/assets/javascripts/user.js]: space before {

**Comment by daodat18plus** [comment]
OK

File: app/assets/stylesheets/calendars.css.scss [hunk 4]
```diff
@@ -536,3 +536,135 @@ span.ui-icon.ui-icon-circle-triangle-e {
...
+  color: white;
+  content: "\2714";
+  font-size: 13px;
+  left: 0px;
+  padding: 2px 8px 2px 2px;
+  position: absolute;
+  top: 0px;
+}
+
+
```
Comment (limkimhuor) [inline app/assets/stylesheets/calendars.css.scss]: remove 1 line

File: app/assets/javascripts/calendar.js [hunk 5]
```diff
@@ -437,4 +441,25 @@ $(document).on('page:change', function() {
     $('#full-calendar').fullCalendar('removeEvents');
     $('#full-calendar').fullCalendar('refetchEvents');
   });
+
+  $('.input-assumpte').change(function() {
+    $('input:checkbox[class=input-assumpte]:checked').not(this).prop("checked", false);
+    color_id = $(this).attr('rel');
+    calendar_id = $('#menu-calendar-id').attr('rel');
+    url = '/api/calendars/' + calendar_id
+    console.log(url);
```
Comment (limkimhuor) [inline app/assets/javascripts/calendar.js]: remove console.log

**Pull request merged**